### PR TITLE
Adapt Monitoring Che for downstream usage

### PR DESCRIPTION
**Pull request by themr0c**
Make references to Grafana conditionnal.

### CODE REVIEW

File: src/main/pages/che-7/administration-guide/proc_collecting-che-metrics-with-prometheus.adoc [hunk 1]
```diff
@@ -31,18 +33,10 @@ histogram:: a more complex metric that is suited for performing observations. Me
...
+            - targets: ['{prod-host}:8087']
   kind: ConfigMap
   metadata:
     name: prometheus-config
 ----
 <1> rate, at which a target is scraped
 <2> rate, at which recording and alerting rules are re-checked (not used in our system at the moment)
-
-
-// [discrete]
```
Comment (MichalMaler) [inline src/main/pages/che-7/administration-guide/proc_collecting-che-metrics-with-prometheus.adoc]: ////
.Additional resources
 
* A bulleted list of links to other material closely related to the contents of the procedure module.
* For more details on writing procedure modules, see the link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
* Use a consistent system for file names, IDs, and titles. For tips, see _Anchor Names and File Names_ in link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
////

Comment (themr0c) [reply src/main/pages/che-7/administration-guide/proc_collecting-che-metrics-with-prometheus.adoc]: ???

Comment (themr0c) [reply src/main/pages/che-7/administration-guide/proc_collecting-che-metrics-with-prometheus.adoc]: I remove comments that are remains from the template.

File: src/main/pages/che-7/administration-guide/proc_collecting-che-metrics-with-prometheus.adoc [hunk 2]
```diff
@@ -31,18 +33,10 @@ histogram:: a more complex metric that is suited for performing observations. Me
...
-// == Additional resources
-// 
-// * A bulleted list of links to other material closely related to the contents of the procedure module.
-// * For more details on writing procedure modules, see the link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
-// * Use a consistent system for file names, IDs, and titles. For tips, see _Anchor Names and File Names_ in link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
```
Comment (MichalMaler) [inline src/main/pages/che-7/administration-guide/proc_collecting-che-metrics-with-prometheus.adoc]: //// 
.Additional resources
 
 * A bulleted list of links to other material closely related to the contents of the procedure module.
 * For more details on writing procedure modules, see the link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
 * Use a consistent system for file names, IDs, and titles. For tips, see _Anchor Names and File Names_ in link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
////

**Review by slemeur — APPROVED** [review]

**Pull request merged**